lib/nerve.rb

```diff
@@ -30,6 +30,7 @@ def initialize(opts={})
       @instance_id = opts['instance_id']
       @services = opts['services']
       @heartbeat_path = opts['heartbeat_path']
+      @affinity = opts['affinity']
```

Comment:
I think `affinity` should be a per-service configuration instead of a global configuration.

The reason is that Nerve has multiple ways of deployments:
  - co-located with the service it's checking. In this case the box that's running both Nerve and the service is the entity that we should set affinity to
  - Nerve checks a remote service's healthiness. In this case the box that's running Nerve is not the same box that's running the service Nerve's checking. In this case affinity should be set to the box that's running the service, not that of Nerve